geocoder/geocoder.cpp

```diff
@@ -156,6 +156,13 @@ void Geocoder::Context::FillResults(vector<Result> & results) const
     results.emplace_back(e.m_key /* osmId */, e.m_value /* certainty */);
   }
 
+  if (!results.empty())
```

Comment:
No need in if.

Reply:
The next statement uses is results.front().

Reply:
I though the branchless version is less readable:
```
for (int i = static_cast<int>(results.size()) - 1; i >= 0; i--)
  results[i].m_certainty /= results[0].m_certainty;
```

Reply:
Okay.